CONTRIBUTING.rst

```diff
@@ -101,14 +101,15 @@ Before you submit a pull request, check that it meets these guidelines:
 2. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a docstring, and add the
    feature to the list in README.rst.
-3. The pull request should work for Python 2.6, 2.7, and 3.3, and for PyPy. Check 
+3. The pull request should pass the Continuous Integration tests. Check
    https://travis-ci.org/dyve/django-bootstrap4/pull_requests
-   and make sure that the tests pass for all supported Python versions.
+   and make sure that all tests pass. You can run the tests locally
+   using `make tox`.
```

Comment:
`make tox` is less platform independent compared to just running `tox` (which should also work on windows).

Reply:
Agreed. Fuxed.